.golangci.yml

```diff
@@ -6,6 +6,8 @@ linters:
     - gochecknoinits
     - interfacer # author says it's deprecated, gives very subjective advice
     - prealloc
+    - scopelint # rhs: this is either incomprehensible or buggy
+    - unparam # rhs: this considers it an error to have a function that always receives the same parameter
```

Comment:
I wrote 15 days ago:
> Fun fact: the author of unparam used to be a Parabola developer.

Reply:
I wrote 15 days ago:
> I'd tend to prefer to add `// nolint` comments or even set up an `exclude-rule` in `.golangci.yml` than to disable `unparam` everywhere, since it is usually useful IMO.

Reply:
I think a good rule for `unparam` is

```yaml
  exclude-rules:
    # unparam complains if a function is always called with the same
    # argument.  That's a silly thing to check for.
    - linters: [unparam]
      text: "^.* always receives `.*`$"
```

Reply:
This doesn't seem to trigger anymore, so for now I've re-enabled the unparam linter. If it crops up again I will try the exclude-rule.